### ISSUE TITLE
Teleporter broken?

### ISSUE THREAD

**Issue by Omanoctoa** [marcelo-mason/PreciousStones#1069]
Hello! I've been trying to work out teleport flags to create a simple, player-oriented teleportation mechanic. Basically, players place 2 fields and give them the same names, then can sneak within that field to teleport. Here's a snippet...

`
- title: Teleporter
  meta-name: '&6Teleporter'
  meta-lore:
  - '&3Teleports you to another field of the same name.'
  - '&7Use &e/ps setname ... &7to name your field,'
  - '&7then place a second one with the same name.'
  meta-autoset: false
  ^teleport-on-sneak: true
  teleport-destination: true
  block: end_stone
  radius: 3
  custom-height: 5
  cuboid: true
  breakable: false
  required-permission: perks.survival.teleporter
  no-conflict: true
  mixing-group: 5
`

The fields place and can be named no problem, but do nothing. Any thoughts? To me understanding this should work as is...but isn't. I've tried it with and without the global ^teleport-on-sneak flag, and yes, I have permission :)

**Comment by krustymk** [comment]
Pretty sure the destination and teleport  have to be on separate stones

**Comment by Omanoctoa** [comment]
On the Field Flag Reference, it says teleport and destination can be on the same teleport field, giving it two-way abilities. I've searched around for more documentation on the teleport flags but haven't found anything on them.

**Comment by Omanoctoa** [comment]
Still not working. Did a test with a few other flags, etc.
```
enable-with-redstone: true
teleport-players-on-enable: true
teleport-if-walking-on: [22, 123, 124, 157]
```
Added these 3 and tested in a variety of ways. Debug shows ```[Server] INFO Omanoctoa entered a Teleporter field``` but apart from me renaming both fields to the same name, nothing else appears in console.

I considered trying to hook the field into WorldGuard Anchors, and using WG to teleport players to an anchor they create when they place the field, but I haven't figured out how to remove the Anchors when they break the field, or how to specify the teleport location. I'd prefer not to use this method...

Help?

**Comment by Omanoctoa** [comment]
After a week of trying to figure this out, I found out why it wasn't working. It had nothing to do with the configuration or the flags. Apparently by giving myself and any players all the permissions...I also gave them ```preciousstones.bypass.teleport``` which disables teleporters for that player. In order to make it work, I had to either remove that perm or negate it (for myself).

So, mostly solved! Now i just have to figure out why it only partially teleports me!
